tests/integration/flows/test_adls_to_azure_sql.py

```diff
@@ -67,3 +70,35 @@ def test_df_to_csv_task_none(caplog):
     task.run(df, path=path, remove_tab=False)
     assert "DataFrame is None" in caplog.text
     assert os.path.isfile(path) == False
+
+
+@pytest.mark.dtypes
+def test_check_dtypes_sort():
+    d = {"col1": ["rat", "cat"], "col2": [3, 4]}
+    df = pd.DataFrame(data=d)
+    dtypes = {
+        "col1": "varchar(6)",
+        "col2": "varchar(6)",
+    }
+    task = check_dtypes_sort
+    n_dtypes = task.run(df=df, dtypes=dtypes)
+    assert list(dtypes.keys()) == list(n_dtypes.keys())
+
+    dtypes = {
+        "col2": "varchar(6)",
+        "col1": "varchar(6)",
+    }
+    task = check_dtypes_sort
```

Comment:
Do you have to overwrite the task object every time? Maybe once on line 83, then just  run it.

Reply:
You are right, changed!